packages/math-input/src/components/keypad/shared-keys.tsx

```diff
@@ -37,21 +37,21 @@ export default function SharedKeys(props: Props) {
     return (
         <>
             <KeypadButton
-                keyConfig={Keys.PLUS}
+                keyConfig={Keys.FRAC}
                 onClickKey={onClickKey}
-                coord={[4, 0]}
+                coord={fractionCoord}
                 secondary
             />
             <KeypadButton
-                keyConfig={Keys.MINUS}
+                keyConfig={Keys.PLUS}
                 onClickKey={onClickKey}
-                coord={[5, 0]}
+                coord={[4, 0]}
                 secondary
             />
             <KeypadButton
-                keyConfig={Keys.FRAC}
+                keyConfig={Keys.MINUS}
                 onClickKey={onClickKey}
-                coord={fractionCoord}
+                coord={[5, 0]}
```

Comment:
So the order these show up in the React/HTML DOM is what defines the tab order, right?

Reply:
Yup!